hil-test/tests/rmt.rs

```diff
@@ -6,120 +6,204 @@
 #![no_std]
 #![no_main]
 
+use core::fmt::Debug;
+
 use esp_hal::{
-    gpio::Level,
-    rmt::{PulseCode, Rmt, RxChannel, RxChannelConfig, TxChannel, TxChannelConfig},
+    gpio::{Level, NoPin},
+    rmt::{Error, PulseCode, Rmt, RxChannel, RxChannelConfig, TxChannel, TxChannelConfig},
     time::Rate,
 };
 use hil_test as _;
 
+cfg_if::cfg_if! {
+    if #[cfg(feature = "esp32h2")] {
```

Comment:
I believe we define the metadata symbols for hil tests, so you should be able to use `cfg(esp32h2)` in tests as well. You don't have to, though, this is fine as is.

Reply:
Indeed, they are defined. If I remember correctly, this was using the `feature = ...` form before, and I just kept it. But it's an easy enough change to add here; I'll do so :+1: